src/tests/Interop/COM/ComWrappers/API/Program.cs

```diff
@@ -188,9 +189,14 @@ public void ValidateComInterfaceCreationRoundTrip()
             var testObjUnwrapped = wrappers.GetOrCreateObjectForComInstance(comWrapper, CreateObjectFlags.Unwrap);
             Assert.Same(testObj, testObjUnwrapped);
 
+            // UniqueInstance and Unwrap should always be a new com object, never unwrapped
+            var testObjUniqueUnwrapped = wrappers.GetOrCreateObjectForComInstance(comWrapper, CreateObjectFlags.Unwrap | CreateObjectFlags.UniqueInstance);
+            Assert.NotSame(testObj, testObjUniqueUnwrapped);
+
             // Release the wrapper
             int count = Marshal.Release(comWrapper);
-            Assert.Equal(0, count);
+            // Expect 1 reference remaining from UniqueInstance
+            Assert.Equal(1, count);
```

Comment:
I would revert this change and use the `ComWrappers.FinalRelease()` to reduce that reference count.

Tests that validate the count drops to 0 are really important so we ensure we create stable and predictable systems.

Reply:
Added a FinalRelease on the ITestObjectWrapper to do this.